packages/api/blocks-api/block-meta.json

```diff
@@ -1,4 +1,185 @@
 [
+    {
+        "name": "Anchor",
```

Comment:
This change seems wrong to me. Most of the blocks that are added here aren't in `@comet/blocks-api`. Could you please generate it one more time?

Reply:
will do. Thanks for the hint 👍🏻

Reply:
https://github.com/vivid-planet/comet/pull/2068/commits/53ff58c92829f388b1f2ef36b0081a3b9f3f4a19